outport/factory/outportFactory.go

```diff
@@ -0,0 +1,86 @@
+package factory
+
+import (
+	logger "github.com/ElrondNetwork/elrond-go-logger"
+	"github.com/ElrondNetwork/elrond-go/core"
+	"github.com/ElrondNetwork/elrond-go/data"
+	"github.com/ElrondNetwork/elrond-go/epochStart"
+	"github.com/ElrondNetwork/elrond-go/epochStart/notifier"
+	"github.com/ElrondNetwork/elrond-go/outport"
+	driversFactory "github.com/ElrondNetwork/elrond-go/outport/drivers/factory"
+	"github.com/ElrondNetwork/elrond-go/sharding"
+)
+
+var log = logger.GetOrCreate("outport/factory")
+
+// ArgsOutportFactory is structure used to store all components that are needed to create a new instance of
+// DriverHandler
+type ArgsOutportFactory struct {
+	ArgsElasticDriver  *driversFactory.ArgsElasticDriverFactory
+	EpochStartNotifier sharding.EpochStartEventNotifier
+	NodesCoordinator   sharding.NodesCoordinator
+}
+
+// CreateOutport will create a new instance of OutportHandler
+func CreateOutport(args *ArgsOutportFactory) (outport.OutportHandler, error) {
+	outportHandler := outport.NewOutport()
+
+	shardID := args.ArgsElasticDriver.ShardCoordinator.SelfId()
```

Comment:
Encapsulate the following 4-5 lines to a new function, if possible? (`registerEpochStartHandler`?)

Reply:
done.